cluster/gce/config-default.sh

```diff
@@ -305,6 +305,8 @@ if [[ "${ENABLE_POD_SECURITY_POLICY:-}" == "true" ]]; then
   ADMISSION_CONTROL="${ADMISSION_CONTROL},PodSecurityPolicy"
 fi
 
+ADMISSION_CONTROL="${ADMISSION_CONTROL},MutatingAdmissionWebhook,ValidatingAdmissionWebhook"
```

Comment:
It might make sense to combing 308 with 311 to minimize the chance that anything is placed between the Validating webhook and ResourceQuota.

Reply:
It's alright if something is placed in between. See https://github.com/kubernetes/kubernetes/pull/57850#discussion_r168568668.